mage/changelog.md

```diff
@@ -4,6 +4,21 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.5 - Dec 20, 2022
+
+### Major Features and Improvements
+
+- Added support for ancestors, descendants, topological sort, connecting nodes, and chaining nodes.
+  [#180](https://github.com/memgraph/mage/pull/180)
+- Aligned C++ API with Memgraph 2.5
+  [#184](https://github.com/memgraph/mage/pull/184)
+- Made Graph Coloring output vertices and integers instead of strings
```

Comment:
```suggestion
- Graph Coloring no longer outputs strings but vertices and integers.
```
Why is this important? what can i do with vertices and integers i couldn't do with strings?

Reply:
You can use them further in Lab and so on.